src/navigation/MainNavigator.tsx

```diff
@@ -13,7 +13,7 @@ export function MainLayout() {
     <MainStack.Navigator
       screenOptions={({ route }) => ({
         tabBarIcon: ({ focused, color, size }) => {
-          let iconName;
+          let iconName: string = '';
 
           if (route.name === 'Search') {
             iconName = focused ? 'search' : 'search-outline';
```

Comment:
You could use a switch instead of if-else in this case to improve code readability

Reply:
thanks, I have changed it with a ternary operator and an object